CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ Changelog for NeoFS Node
 ### Added
 - Serving `NetmapService.NetmapSnapshot` RPC (#1793)
 - `netmap snapshot` command of NeoFS CLI (#1793)
-
+- `apiclient.allow_external` config flag to fallback to node external addresses (#1817)
```

Comment:
do we need to add that to the `Update from...` section too?

Reply:
I think no, it is a completely new feature, old nodes are expected to continue working.

Reply:
yes, but how will an admin know about that? i try to write any config changes to that section. but ok

Reply:
I suggest adding this to Update section. With new behaviour node admins can explicitly declare external addresses for their node, not only listen corresponding addresses.

Reply:
Yes, but do we need `Added` section in this case? Every added feature can be used in some way, I've though `Updating from` was for things requiring special care.
Fixed anyway.

Reply:
If we need Ops attention, we add/duplicate something in `Updating from`. For this particular case we need to highlight it.